humble_gz_macos copy.repos

```diff
@@ -0,0 +1,37 @@
+repositories:
+  ardupilot:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot.git
+    version: master
+  ardupilot_gazebo:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot_gazebo.git
+    version: ros2
+  ardupilot_gz:
+    type: git
+    url: https://github.com/ArduPilot/ardupilot_gz.git
+    version: main
+  ardupilot_sitl_models:
+    type: git
+    url: https://github.com/ArduPilot/SITL_Models.git
+    version: main
+  gps_umd:
+    type: git
+    url: https://github.com/swri-robotics/gps_umd.git
+    version: ros2-devel
+  micro_ros_agent:
+    type: git
+    url: https://github.com/srmainwaring/micro-ROS-Agent.git
+    version: jazzy-macos
```

Comment:
@srmainwaring I don't know if macOS will need a separate branch for `Jazzy`. Can you check/test when possible ?